examples/app-with-dashboards/src/module.ts

```diff
@@ -0,0 +1,12 @@
+import { AppPlugin } from '@grafana/data';
+import { App } from './components/App';
+import { AppConfig } from './components/AppConfig';
+
+export const plugin = new AppPlugin<{}>().setRootPage(App).addConfigPage({
+  title: 'Configuration',
+  icon: 'cog',
+  // @ts-ignore - Would expect a Class component, however works absolutely fine with a functional one
```

Comment:
I prefer we don't ship examples with typescript ignore errors.

Reply:
Good catch, we have to update our templates in create-plugin.

I am not sure why it was there, maybe it was needed previously. It seems to build properly without it. 
https://github.com/grafana/grafana-plugin-examples/pull/99/commits/8884b24ee50f3bffa928d463ec1a1447457178f6

and 

https://github.com/grafana/grafana-plugin-examples/pull/99/commits/1e13559b4b5f08c078c117c77d064ca8928899f3

Reply:
https://github.com/grafana/plugin-tools/pull/124